components/uniset/src/props.rs

```diff
@@ -34,263 +34,263 @@ fn get_prop<'data, D: DataProvider<'data, UnicodePropertyV1Marker> + ?Sized>(ppu
 // Binary property getter fns
 //
 
-pub fn get_ascii_hex_digit_property<'data, D: DataProvider<'data, UnicodePropertyV1Marker> + ?Sized>(provider: &D) -> UnisetResult {
+pub fn get_ascii_hex_digit_property<'data, D: DataProvider<'data, UnicodePropertyV1Marker> + ?Sized>(provider: &'data D) -> UnisetResult {
```

Comment:
The reference to the provider should be the default function lifetime, not `'data`.  The `'data` is the lifetime argument of DataProvider, not the lifetime of the reference to it.

Reply:
Follow-up: #1095